projects/admin/src/app/acquisition/components/account/account-list/account-list.component.ts

```diff
@@ -63,4 +101,57 @@ export class AccountListComponent implements OnInit {
     this.rootAccounts = this.rootAccounts.filter(item => item.pid !== account.pid);
   }
 
+  /**
+   * Get Export formats for the current resource given by configuration.
+   * @return Array of export format to generate an `export as` button or an empty array.
+   */
+  private _exportFormats(): Array<any> {
+    if (exportFormats) {
+      return exportFormats.map(
+        (format) => {
+          return {
+            label: format.label,
+            url: this.getExportFormatUrl(format),
+            disabled: !this.canExport(format),
+            disabled_message: this.exportInfoMessage
+          };
+        }
+      );
+    }
+    return [];
```

Comment:
reverting the `if` condition will saves some indentation spaces.
```javascript
if (exportFormat === undefined || exportFormat.length === 0) {
    return [];
}
```
Maybe using a default "empty list" value on `exportFormats` attribute will save you from this condition.

Reply:
done